<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anime Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="8">
     <widget class="QPushButton" name="findButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>Delete search results</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="5">
     <widget class="QLineEdit" name="searchBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="19" column="10">
     <widget class="QSpinBox" name="deleteIdx">
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>0</height>
       </size>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="11" column="10" colspan="2">
     <widget class="QSpinBox" name="spinBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="loadButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="11">
     <widget class="QLabel" name="count">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="5">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Choose genres:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QPushButton" name="showAllButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Show all</string>
      </property>
     </widget>
    </item>
    <item row="11" column="9">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Page count:</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="12">
     <widget class="QTableView" name="tableView">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Search:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="9">
     <widget class="QPushButton" name="deleteRowButton">
      <property name="text">
       <string>Delete Row</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="sortButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Sort by rating</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save to file</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionAll"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuSave"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Current table</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
